bounties_api/std_bounties/client_helpers.py

```diff
@@ -38,11 +38,17 @@
 
 def map_bounty_data(data_hash, bounty_id):
     ipfs_hash = data_hash
-    if len(ipfs_hash) != 46 or not ipfs_hash.startswith('Qm'):
-        logger.error('Data Hash Incorrect for bounty: {:d}'.format(bounty_id))
-        data_JSON = "{}"
-    else:
-        data_JSON = ipfs.cat(ipfs_hash)
+    try:
+        if len(ipfs_hash) != 46 or not ipfs_hash.startswith('Qm'):
+            logger.error('Data Hash Incorrect for bounty: {:d}'.format(bounty_id))
+            data_JSON = "{}"
+        else:
+            data_JSON = ipfs.cat(ipfs_hash)
+    except StatusError as e:
```

Comment:
If these two `except`s are not in the form of
> StatusError: HTTPError: 504 Server Error: Gateway Time-out for url

are we still happy to blacklist them this way? If we got a different error back, it would still make sense to blacklist it this way, in my opinion (with limited knowledge).